### ISSUE TITLE
Does not parse arguments on ubuntu linux

### ISSUE THREAD

**Issue by mostalive** [ThomasCrevoisier/psvm-js#23]
I've made a fix, posted a pull request at cliparse-node, but it does not seem very active.

https://github.com/divarvel/cliparse-node/pull/19

Also tested locally, and it works for me, on my machine ;-).

I hope the pull request gets merged, but not too optimistic (a few others in the queue there).

**Comment by hdgarrood** [comment]
thanks for tracking this down! This affected me too, but I couldn't work out what was going on. It might also be worth considering switching to a more popular package for getting command line arguments, such as [yargs](https://github.com/yargs/yargs)?

**Comment by mostalive** [comment]
I've sent the mainainer an e-mail, maybe I'll get some more info about the
status of it. I'll also ask a friend who lives locally and is probably
active in the same communities.

I tracked it down after hesitating at least three times (I'd prefer Stack
for Purescript, but can't afford to build it anytime soon). Change was
surprisingly easy to make, thanks to the issue I found describing the
location already. Test suite is not too bad either, so I'm moderately
confident it didn't break e.g. the windows version.

Upsides of switching:
- better maintained

Downsides of switching
- rewrite costs time and effort
- well maintained may mean more time spent on breaking things because the
  API changes and its not _that_ important.
